changelogs/fragments/213-read-write-only.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - "api_info - add new ``include_read_only`` option to select behavior for read-only values. By default these are not returned (https://github.com/ansible-collections/community.routeros/pull/213)."
```

Comment:
The documentation says `default: true` for `include_read_only`. I think the word `not` should be dropped here.

Reply:
I switched everything to `false` except the documentation (that's also why the sanity test fails); I've updated that one instead.

Or do you think it's better to have the default as `true`? I was thinking about this a bit, and eventually decided to use the same default for all `include_*` options. But for this one, it could also make sense to have `true` as the default.

Reply:
I thought including read-only values by default is the more sensible choice for `api_info`, but you have a point with consistency with the other `include_*` options. Let's leave it as false.